code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -181,3 +181,75 @@
 	if(!(target.status_flags & GODMODE) && target.stat != DEAD)
 		user.adjustBruteLoss(-force*0.2)
 	..()
+
+/obj/item/ego_weapon/gold_rush
+	name = "gold rush"
+	desc = "The weapon of someone who can swing their weight around like a truck"
+	icon_state = "gold_rush"
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
+	var/goldrush_damage = 130
+
+//Replaces the normal attack with the gigafuck punch
+/obj/item/ego_weapon/gold_rush/attack(mob/living/target, mob/living/user)
+	if(!CanUseEgo(user))
+		return
+	if(do_after(user, 4, target))
+
+		target.visible_message("<span class='danger'>[user] rears up and slams into [target]!</span>", \
+						"<span class='userdanger'>[user] punches you with everything you got!!</span>", COMBAT_MESSAGE_RANGE, user)
+		to_chat(user, "<span class='danger'>You throw your entire body into this punch!</span>")
+
+
+		//I gotta regrab  justice here
+		var/userjust = (get_attribute_level(user, JUSTICE_ATTRIBUTE))
+		var/justicemod = 1 + userjust/100
+		goldrush_damage *= justicemod
+
+		if(ishuman(target))
+			goldrush_damage = 50
+
+		target.apply_damage(goldrush_damage, RED_DAMAGE, null, target.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)		//MASSIVE fuckoff punch
+
+		playsound(src, 'sound/weapons/ego/paradise_ranged.ogg', 50, TRUE)
```

Comment:
But why.

Reply:
I forgot to change the sound path :skull: